pkg/backend/httpstate/client/client.go

```diff
@@ -536,18 +552,70 @@ func (pc *Client) PublishPolicyPack(ctx context.Context, orgName string,
 	return nil
 }
 
-// ApplyPolicyPack applies a `PolicyPack` to the Pulumi organization.
-func (pc *Client) ApplyPolicyPack(ctx context.Context, orgName string, policyPackName string,
-	version int) error {
+// ApplyPolicyPack enables a `PolicyPack` to the Pulumi organization. If policyGroup is not empty,
+// it will enable the PolicyPack on the default PolicyGroup.
+func (pc *Client) ApplyPolicyPack(ctx context.Context, orgName string, policyGroup string,
+	policyPackName string, version int) error {
+
+	if policyGroup == "" {
+		req := apitype.ApplyPolicyPackRequest{Name: policyPackName, Version: version}
+
+		err := pc.restCall(
+			ctx, "POST", applyPolicyPackPath(orgName, policyPackName, version), nil, req, nil)
+		if err != nil {
+			return errors.Wrapf(err, "HTTP POST to enable policy pack failed")
```

Comment:
Is the "HTTP POST to " relevant to the error? Perhaps we can omit it for clarity?

Reply:
Was following the previous established pattern, but I agree its not needed. Will edit all.